mne/epochs.py

```diff
@@ -819,9 +819,14 @@ def _reject_setup(self, reject, flat):
         # check for invalid values
         for rej, kind in zip((reject, flat), ("Rejection", "Flat")):
             for key, val in rej.items():
-                if val is None or val < 0:
+                if callable(val):
```

Comment:
I would simplify to:
```
if callable(val):
    continue
else:
    name = f"{kind} dict value for {key}"
    _validate_type(val, "numeric", name, extra="or callable")
    if val < 0:
         raise ValueError("{kind} {name} must be >= 0, got {val}")
```

Reply:
I was going to do it this way, but it breaks a test. I think its because `_validate_type `returns a TypeError and Pytest wants a ValueError:
```
   for val in (None, -1):  # protect against older MNE-C types
        for kwarg in ("reject", "flat"):
            pytest.raises(
                ValueError,
                Epochs,
                raw,
                events,
                event_id,
                tmin,
                tmax,
                picks=picks_meg,
                preload=False,
                **{kwarg: dict(grad=val)},
            )
```

Reply:
We should just update the test, TypeError is a better error in the case where the type is wrong (i.e., `_validate_type` does something better than what we do in `main`)